src/lib/eina/eina_sched.c

```diff
@@ -0,0 +1,84 @@
+/* EINA - EFL data type library
+ * Copyright (C) 2010 ProFUSION embedded systems
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library;
+ * if not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifdef HAVE_CONFIG_H
+# include "config.h"
+#endif
+
+#include <pthread.h>
+#ifdef __linux__
+# include <sched.h>
+# include <sys/time.h>
+# include <sys/resource.h>
+# include <errno.h>
+#endif
+
+#include "eina_sched.h"
+#include "eina_log.h"
+
+#define RTNICENESS 1
+#define NICENESS 5
+
+void
+eina_sched_prio_drop(void)
+{
+   struct sched_param param;
+   int pol, ret;
+   pthread_t pthread_id;
```

Comment:
Won't this be a problem?

Reply:
`eina_sched.c` is linux only, the implementation for windows is inside `eina_thread_win32.c`

Reply:
Oh wait, every sched function on win32 is on eina_thread_win32? phab is ok with that? I think if we are doing alternative functions to posix implementation we should have `_posix/_win32` files

Reply:
I didn't touch sched in upstream yet, so this is the temporary solution to compile in native-windows without departing the implementation too much from phab.

Reply:
ohh understood, as it isn't the final solution it's okay then